utils/log/logger.go

```diff
@@ -18,13 +19,24 @@ type LogFormat string
 // Used for coloring sections of the log message. For example log.Format.Path("...")
 var Format LogFormat
 
-// Determines whether the terminal is available. This variable should not be accessed directly (besides  tests),,
-// but through the 'isTerminal' function.
-var TerminalMode *bool
+// Determines whether the Stdout is terminal is available. This variable should not be accessed directly,
+// but through the 'IsTerminal' function.
+var StdOutIsTerminal *bool
+
+// Determines whether the Stderr is  terminal is available. This variable should not be accessed directly,
+// but through the 'IsTerminal' function.
+var StdErrIsTerminal *bool
```

Comment:
Shound't this variable be private?

Reply:
@Or-Geva needs them to be public to manipulate them from outside on his cli tree view tests